`@types/stack-trace` is missing from dependencies.

I'm getting the following error when trying to build from this package:

```terminal
node_modules/youch/index.d.ts:2:30 - error TS7016: Could not find a declaration file for module 'stack-trace'. '/Users/peterp/x/redwoodjs/redwood/node_modules/stack-trace/lib/stack-trace.js' implicitly has an 'any' type.
  Try `npm install @types/stack-trace` if it exists or add a new declaration (.d.ts) file containing `declare module 'stack-trace';`
```

--- a/package.json
+++ b/package.json
@@ -36,7 +36,8 @@
   "dependencies": {
     "cookie": "^0.4.1",
     "mustache": "^4.0.1",
-    "stack-trace": "0.0.10"
+    "stack-trace": "0.0.10",
+    "@types/stack-trace": "^0.0.29"
   },
   "repository": {
     "type": "git",

